src/Card/CardFooter.jsx

```diff
@@ -0,0 +1,46 @@
+import React, { useContext } from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import CardContext from './CardContext';
+
+const CardFooter = React.forwardRef(({
+  children,
+  className,
+  isStacked,
+  textElement,
+  orientation,
+}, ref) => {
+  const { orientation: cardOrientation } = useContext(CardContext);
+  const footerOrientation = orientation || cardOrientation;
+  const wrapperClassName = `pgn__card-footer ${footerOrientation}${isStacked ? '-stacked' : ''}`;
+  const textElementClassName = `pgn__card-footer-text ${footerOrientation}${isStacked ? '-stacked' : ''} x-small`;
```

Comment:
Let's move use of `x-small` CSS utility class to a SCSS file with SCSS variable so that we aren't unintentionally using an `!important` (which limits theming)

Reply:
Agree, this should be better, moved it in last commit